gitbot/lib.py

```diff
@@ -169,8 +169,16 @@ def bump_version(
         run(f"git config user.name {COMMITTER_NAME}", cwd=repo_root)
         run(f"git config user.email {COMMITTER_EMAIL}", cwd=repo_root)
 
-        command = bump_command(ref_sha, author)
-        run(command, cwd=repo_root)
+        try:
+            command = bump_command(ref_sha, author)
+            run(command, cwd=repo_root)
+        except CommandError:
+            execution = run("git show", cwd=repo_root)
+            # e.g. https://github.com/getsentry/getsentry/pull/7672/commits
+            if execution.stdout.find(f"getsentry/sentry@{ref_sha}") > -1:
```

Comment:
This doesn't happen often but I would rather not get anyone distracted:
https://sentry.io/organizations/sentry/issues/2677689153/?environment=production&project=5748916&query=&statsPeriod=90d